test/CMakeLists.txt

```diff
@@ -986,6 +992,9 @@ COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --ver
 COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --verbose --half --input  1 1 8 8 --weights 1 1 2 2 --pads_strides_dilations 0 0 1 1 2 2 --disable-forward --disable-backward-data
 COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --verbose --half --input  1 128 56 56 --weights 1 128 5 5 --pads_strides_dilations 0 0 2 2 1 1 --disable-forward --disable-backward-data
 )
+add_custom_test(test_conv_igemm_ck_xdlops_bwd_v4r1 SKIP_UNLESS_ALL HALF_ENABLED GFX908_ENABLED VEGA_DISABLED
+COMMAND ${CK_IMPLICITGEMM_BWD_V4R1_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --verbose --half --input  128  24 14 14 --weights 64  24 5 5 --pads_strides_dilations 2 2 1 1 1 1 --disable-forward --disable-backward-weights
```

Comment:
`--half` must be the first option (otherwise ignored), pls see https://github.com/ROCmSoftwarePlatform/MIOpen/issues/990#issuecomment-864467122